src/components/counterComponent/counter.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import { Counter } from './counter';
+
+test('test counter component', () => {
+  render(<Counter initialValue='0' />);
+  const counterValue = screen.getByTestId("counter-value");
+  expect(counterValue).toHaveTextContent(0);
+});
```

Comment:
There is scope to add more unit tests for each components. 

For example, for this Counter component, a test to test the increment button functionality and another to test the decrease button functionality.